conjur/argument_parser/_hostfactory_parser.py

```diff
@@ -172,35 +189,44 @@ def _add_hostfactory_create_token(menu: ArgparseWrapper):
             title=title_formatter("Options"))
         create_token.add_argument('-action_type', default='create_token', help=argparse.SUPPRESS)
         create_token.add_argument('-i', '--hostfactoryid', metavar='VALUE', required=True,
+                                    #Needs TW review
                                   help='(Mandatory) The ID of the Host Factory'
                                         'you are working work with.')
         create_token.add_argument('--cidr', metavar='VALUE',
+                                    #Needs TW review
                                   help='(Optional) the CIDR address that contains '
```

Comment:
(Optional) CIDR containing all IP addresses that can use Host Factory token for creating hosts